docs/source/apis/saas_api.rst

```diff
@@ -0,0 +1,151 @@
+
+   - There will be an initiation fee that will be collected upfront for all new users. Please refer to the pricing information page on our website.
+7. Click the "Pay this invoice" button in the email and follow the instructions to enter your payment details securely on Stripe website.
+8. Confirm the payment to complete this process. Look out for any confirmation prompts or messages that signify a successful transaction.
+
+   - Note that we don't send out a receipt. If you want to keep a receipt, click the "Download receipt" button after completing the payment on Stripe.
+
+
+Using the API Key and URL
+--------------------------
+
+1. After your payment is processed, you will receive an email from us containing your unique API credentials.
+2. Check your inbox (and the spam folder, just in case) for an email from our domain.
+3. Save these credentials securely, as you will need them to access and integrate our API with your applications.
+
+
+Update Billing
+--------------
+
+If you need to update your billing information, fill out the "Update Billing" form.
+
+1. Select "Update Billing" on `Unstructured SaaS API <https://unstructured.io/api-key-hosted>`_ webpage
+2. Upon successful sign-up, you will be redirected to the Stripe payment page.
+3. Follow the instructions to enter your payment details securely.
+4. Confirm the payment to complete this process. Look out for any confirmation prompts or messages that signify a successful transaction.
+
+.. image:: imgs/paid_api/update_billing.png
+  :align: center
+  :alt: Update Billing
+
+
+Opt-in to Marketing and Usage Emails
+------------------------------------
+
+To receive regular updates on usage statistics and marketing emails, please follow these steps:
+
+1. Select "Subscribe to Usage Updates" on `Unstructured SaaS API <https://unstructured.io/api-key-hosted>`_ webpage.
+2. Enter your email address in the provided field.
+3. Click the "Submit" button to complete the subscription process.
+
+You will receive a confirmation email shortly after submitting your information. Please click the confirmation link to complete your opt-in to usage and marketing emails.
+
+.. image:: imgs/paid_api/opt-in.png
+  :align: center
+  :alt: Opt-in Email
+
+
+Rotate a New API Key
+--------------------
+
+If you need to rotate your API key for security reasons or API key regeneration, follow these steps:
+
+1. Select "Get a new API Key" form on `Unstructured SaaS API <https://unstructured.io/api-key-hosted>`_ webpage.
+2. Type your registered *Email* address in the corresponding input field.
+3. Provide your *Current API Key* in the specified field.
+4. Press the *Submit* button to initiate the API key rotation process.
+
+Once you submit the form, our system automatically generates a new API key and deactivates the old one. You will receive the new API key at your registered email address. Be sure to update your applications with the new API key to maintain access to the API services.
```

Comment:
@treighton and @amanda103 Could you confirm if the user will receive a confirmation email after submitting the API key rotation form?